draft-ietf-ppm-dap.md

```diff
@@ -505,18 +503,18 @@ parameters used by the Aggregators to select a batch of reports specific to the
 given query type. A query is defined as follows:
 
 ~~~
+uint64 BatchID;
+
 enum {
-    next-batch(0),
-    by-batch-id(1),
+    by-batch-id(0),
+    current-batch(1),
 } FixedSizeQueryType;
 
-opaque BatchId[32];
-
 struct {
-    FixedSizeQueryType fixed_size_query_type;
-    select (fixed_size_query_type) {
-        next-batch:  Empty;
+    FixedSizeQueryType query_type;
+    select (query_type) {
         by-batch-id: BatchId batch_id;
+        current-batch: Empty;
```

Comment:
Won't the line above this need to be `BatchID`?

Reply:
Indeed -- updated.